question.js

```diff
@@ -0,0 +1 @@
+let Question = "we are learning how to answere on the client's comment"
```

Comment:
#Specific line comment3: Why you added this line

Reply:
for Question